cdap-ui/README.rst

```diff
@@ -1,4 +1,108 @@
 ==============
+CDAP UI Pack 5
+==============
+
+The CDAP UI Pack is a UI-only release that can be applied on top of an existing CDAP installation.
+The CDAP UI Pack 5 applies on CDAP 4.2.0.
+
+Details
+=======
+- Release Date: 07/07/2017
+- Base CDAP Version: 4.2.0
+- Release branch: release/4.2
+
+Installation
+============
+
+UNIX/Linux Flavors
+------------------
+**CDAP Sandbox**
+::
+
+  $ cd <CDAP_HOME>
+  $ ./bin/cdap sandbox stop
+  $ ./bin/cdap apply-pack /path/to/download/cdap-ui-pack-4.2.0_p5.zip
+  $ ./bin/cdap sandbox start
+
+
+**Distributed CDAP**
+
+*Note:* In the instructions below, ``<CDAP_HOME>`` is either:
+
+- ``/opt/cdap`` on RPM (manual or Apache Ambari) installations; or
+- the *CDAP* sub-directory of the *Parcel Directory* on Cloudera Manager installations (e.g. ``/opt/cloudera/parcels/CDAP``)
+
+::
+
+  $ cd <CDAP_HOME>
+  $ /etc/init.d/cdap-ui stop
+  $ cdap apply-pack /path/to/download/cdap-ui-pack-4.2.0_p5.zip
+  $ /etc/init.d/cdap-ui start
```

Comment:
general UI is using the Node we ship. I think applying UP is using the system's Node